modules/webhook/main_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package webhook
+
+import (
+	"path/filepath"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+)
+
+func TestMain(m *testing.M) {
+	models.MainTest(m, filepath.Join("..", ".."))
```

Comment:
I don't understand very well how this is supposed to work, but [according to docs](https://golang.org/pkg/testing/#hdr-Main) `os.Exit()` should be called here? Sorry if my question is not relevant.

Reply:
Yeah, I think it works without `os.Exit()`. But I will investigate it.